ServiceWebsite/ServiceWebsite/Controllers/ParticipantController.cs

```diff
@@ -34,8 +39,63 @@ public ParticipantController()
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         public async Task<IActionResult> UpdateSuitabilityAnswers(Guid hearingId, [FromBody]List<HearingSuitabilityAnswer> answers)
         {
-          await Task.Yield();
-          return NoContent();
+            if (hearingId == Guid.Empty)
+            {
+                return new BadRequestResult();
```

Comment:
Include what's bad, an error message in the result so the called knows what's missing.

Reply:
Made changes